docs/api/covidcast_meta.md

```diff
@@ -58,7 +58,7 @@ None.
 | --- | --- | --- |
 | `result` | result code: 1 = success, 2 = too many results, -2 = no results | integer |
 | `epidata` | list of results, one per name/geo_type pair | array of objects |
-| `epidata[].name` | Data souce, and subtype if applicable (e.g. fb_survey_cli, fb_survey_ili) | string |
+| `epidata[].source` | Data souce, and subtype if applicable (e.g. fb_survey_cli, fb_survey_ili) | string |
```

Comment:
i'm fixing the description to just say "data source"

Reply:
thanks! also in this table, do you mind (if you have the time) adding: `time_type`, `min_value`, and `max_value`? also, `min_date` -> `min_time` and `max_date` -> `max_time` (with data type always int, YYYYMMDD for day, YYYYWW for week)

Reply:
fixed